CONTRIBUTING.md

```diff
@@ -84,7 +84,17 @@ To do this from Github's web interface:
 You can help cover domain name registration and renewal costs by pledging a small amount on [Liberapay ![](https://img.shields.io/liberapay/goal/awesome-selfhosted?logo=liberapay) ![](https://img.shields.io/liberapay/receives/awesome-selfhosted?logo=liberapay)](https://liberapay.com/awesome-selfhosted/)
 
 
-### Other
+### Other guidelines
+
+In addition to guidelines listed in the [Pull Request template](.github/PULL_REQUEST_TEMPLATE.md), these general rules help keep the list consistent:
+- Please avoid redundant terms in project descriptions, such as _open-source_, _free_, _self-hosted_... as their presence on awesome-selfhosted already implies this.
+- Prefer shorter forms for descriptions - for example, `Minimalist text adventure game` would be preferred to `A minimalist text adventure game` or `$PROJECT is a minimalist text adventure game`).
+- If the the project has no documentation in English, please add `(documentation in $LANGUAGE)` at the end of the description.
+- If the project is presented as an alternative to another service or application, please mention it as `(alternative to $PRODUCT1, $PRODUCT2)` at the end of the description.
+- If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
+
```

Comment:
```suggestion
- Projects which are forked from other projects must include `(fork of $PROJECT)` in the end of their descriptions.
- Software that distributes a single static binary should indicate the programming language it is written in

```

Reply:
I agree with this as it does make it more clearer to the user on how to handle contributing.